java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/debugging/launch/NbLaunchDelegate.java

```diff
@@ -119,10 +126,22 @@ public void finished(boolean success) {
                     notifyFinished(context, success);
                 }
             };
+            List<String> args = argsToStringList(launchArguments.get("args"));
+            List<String> vmArgs = argsToStringList(launchArguments.get("vmArgs"));
+            
+            List<Object> fixedLookupContents = new ArrayList<>(Arrays.asList(
+                toRun, ioContext, progress
+            ));
+            if (!(args.isEmpty() && vmArgs.isEmpty())) {
+                ExplicitProcessParameters.Builder bld = ExplicitProcessParameters.builder();
+                bld.priorityArgs(vmArgs);
+                bld.args(args);
+                bld.appendArgs(false);
+                fixedLookupContents.add(bld.build());
```

Comment:
It would be useful to describe how to influence Maven/Gradle and pass in the JVM as well as user arguments including the `executeWith` or `ActionProvider` invocation.

Reply:
Documented in 39417f0d8b